course/functions.php

```diff
@@ -42,33 +42,24 @@ function course_support() {
 	 * @return void
 	 */
 	function course_scripts() {
-
-		// Register theme stylesheet.
-		wp_register_style(
-			'course-style',
-			get_stylesheet_directory_uri() . '/style.css',
-			array(),
-			wp_get_theme()->get( 'Version' )
-		);
-
-		wp_register_style(
-			'course-sensei-learning-mode',
-			get_stylesheet_directory_uri() . '/learning-mode.css',
-			array(),
-			wp_get_theme()->get(
-				'Version'
-			)
-		);
-
-		// Enqueue theme stylesheet.
+		wp_register_style( 'course-style', get_stylesheet_directory_uri() . '/style.css', array(), wp_get_theme()->get( 'Version' ) );
+		wp_enqueue_script( 'course-header', get_template_directory_uri() . '/assets/js/header.js', [], wp_get_theme()->get( 'Version' ), true );
 		wp_enqueue_style( 'course-style' );
 
-		// TODO: Only Load it if the the page is using learning mode
-		wp_enqueue_style( 'course-sensei-learning-mode' );
+		$has_sensei = function_exists( 'Sensei' );
```

Comment:
I think we should check if the Sensei dependency is met in the same way as our other plugins - https://github.com/Automattic/sensei-pro/blob/trunk/includes/class-sensei-pro-dependency-checker.php#L107.